entries/l/lucidchart.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Lucidchart": {
+    "domain": "lucidchart.com",
```

```suggestion
    "domain": "lucidchart.com",
    "additional-domains": [
      "lucid.app",
      "lucid.co"
    ],
```
As the authentication seems to be done at these domains, adding them will make sure the listing reaches the most amount of people :wink:

---

good idea, thank you 👍